### DISCUSSION TITLE
File Attributes

### DISCUSSION THREAD

**Discussion by ncwuguo** [sumatrapdfreader/sumatrapdf#4319] [General]
File properties do not show the full directory path of the file, which is important, for some unknown files, I can lock its location very well.Consider dynamically adjusting the number of rows displayed based on the content of file attributes.Otherwise, the function of this file information is meaningless (incomplete display is equivalent to nothing displayed)

Additional, it's font is not required to display such bold.

![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/54211354/838c6d2e-7865-4851-815b-3db30ca60e7a)

**Comment by GitHubRulesOK** [comment]
Filenames could be wider than the screen so there had to be an optimal length otherwise it needs wrapping but if you copy to clipboard you get the full name.
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/f3e951e5-b27c-482d-ac3b-addeb9ed1015)

Also simplest without keyboard you can see the full path by hover over the tab or press F2 or show in folder
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/4f8c55d1-58db-4705-9fa4-5c6c14b132f7)
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/7f5dcfe8-6e4a-46cd-b363-2282a7ea0361)
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/d3f1daa2-d20b-42d5-a711-232f40d33845)
